.env.docker

```
@@ -0,0 +1,30 @@
+DB_PASSWORD="outleads_secure_password"
+
+# Redis
+REDIS_URL="redis://:redis_secure_password@redis:6379"
+REDIS_PASSWORD="redis_secure_password"
+
+# Authentication
+JWT_SECRET="your-super-secret-jwt-key-min-32-chars-change-this"
+NEXTAUTH_SECRET="your-nextauth-secret-key-change-this-too"
+
+# Application
+NODE_ENV="production"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)